Plugins/BuildServerIntegration/AppVeyorIntegration/AppVeyorAdapter.cs

```diff
@@ -185,7 +187,8 @@ private IEnumerable<AppVeyorBuildInfo> QueryBuildsResults(string projectId)
             }
         }
 
-        internal IEnumerable<AppVeyorBuildInfo> ExtractBuildInfo(string projectId, string? result)
+#nullable disable
```

Comment:
Better to use `#nullable disable warnings` here (and elsewhere) so that you can keep annotations in code (eg. line 263).